pkg/util/liveconfig/manager.go

```diff
@@ -21,10 +21,20 @@ type Manager interface {
 	DefaultInstallerPullSpecOverride(context.Context) string
 }
 
-type dev struct{}
+type dev struct {
```

Comment:
shouldn't that struct definition/creation be in dev.go instead? same comment for prod (but in prod.go)

Reply:
Agreed. I never put them there in the first place, but I've refactored both of them.

Reply:
Well now I know why its like that. The mocking is quite tricky with the Azure SDK. In the interest of reducing complexity of this PR I am going to leave it for now and we can do a follow up PR to work on it when time isn't of the essence.